ibis/expr/api.py

```diff
@@ -1300,6 +1314,16 @@ def _integer_to_interval(arg, unit='s'):
 sign = _unary_op('sign', ops.Sign)
 sqrt = _unary_op('sqrt', ops.Sqrt)
 
+# TRIGONOMETRIC OPERATIONS
+acos = _unary_op('acos', ops.Acos)
+asin = _unary_op('asin', ops.Asin)
+atan = _unary_op('atan', ops.Atan)
+atan2 = _generic_op('atan2', ops.Atan2)
```

Comment:
This is a binary operation right? There should be something like `_binary_op` function lying around here.

Reply:
you're right. I think this is the right function: `_binop_expr`